packages/terra-menu/src/MenuItem.jsx

```diff
@@ -113,9 +126,17 @@ class MenuItem extends React.Component {
       if (event.nativeEvent.keyCode === MenuUtils.KEYCODES.ENTER || event.nativeEvent.keyCode === MenuUtils.KEYCODES.SPACE) {
         this.handleSelection(event);
 
+        // If the item has sub menu items onKeyUp will never get called and item will be stuck active
+        if (!this.props.subMenuItems || this.props.subMenuItems.length === 0) {
+          this.setState({ isActive: true });
+        }
+
         if (this.props.onClick) {
           this.props.onClick(event);
         }
+      // Item will become stuck active if the user presses tab while holding space
```

Comment:
Update to `Remove active state when tab key is released while while holding the space key`